package.json

```
@@ -2,8 +2,11 @@
   "name": "@mastodon/mastodon",
   "license": "AGPL-3.0-or-later",
   "engines": {
-    "node": ">=16"
+    "node": ">=18"
```

Seems like it's at 20 in the NVM and Dockerfile
```suggestion
    "node": ">=20"
```

---

But 18 is still supported

---

It is, but only 20 is being tested on and shipped with the images

---

We do not have real tests for the JS part running on Node, ie the streaming server, so this is not really an issue.

We could try to test the Webpack build with both Node 18 and Node 20, but I do not feel this would be very useful, as this is not running our code.